### PR TITLE
Mailchimp tags dest

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+---
+title: Mailchimp Tags
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md:2]: Is Tags supposed to be capitalized? Looking at their website (https://mailchimp.com/help/getting-started-tags/#About_tags), it seems it's all lowercase. Let's clarify if it should be `Mailchimp Tags` or `Mailchimp tags` both in the docs and in the UI.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+---
+# [!DNL Mailchimp Tags] connection
+
+[[!DNL Mailchimp]](https://mailchimp.com) *(also known as [!DNL Intuit Mailchimp])* is a popular marketing automation platform and email marketing service used by businesses to manage and talk to contacts *(clients, customers, or other interested parties)* using mailing lists and email marketing campaigns.
+
+[!DNL Mailchimp Tags] uses [audiences](https://mailchimp.com/help/getting-started-audience/) and [tags](https://mailchimp.com/help/getting-started-tags/) to manage your contact information. Tags are labels using which you can organize your contacts and label them for your internal categorization within [!DNL Mailchimp].
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp Batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) endpoint. You can add contacts or update tags of existing [!DNL Mailchimp] contacts within an existing [!DNL Mailchimp] audience after activating them within a new segment. [!DNL Mailchimp Tags] uses the selected segment names from Platform as the tag names within [!DNL Mailchimp].
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Lower case "batch subscribe"

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+[!DNL Mailchimp Tags] uses [OAuth 2 with Authorization Code](/help/destinations/destination-sdk/functionality/authentication/oauth2-authentication.md#authorization-code) to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Mailchimp Tags] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Mailchimp Tags] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of an organization wants to broadcast an email based marketing campaign to a curated list of contacts. The organizations marketing team can add new contacts or update tags of existing [!DNL Mailchimp] contacts, within an existing [!DNL Mailchimp] audience through Experience Platform, build segments from their own offline data, and send these segments to [!DNL Mailchimp Tags]. This audience can then be used to send the marketing campaign email.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: organization's

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Let's break up the second phrase into multiple sentences. It's rather difficult to follow what you mean there.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that you need to set up in Experience Platform and [!DNL Mailchimp] and for information that you need to gather before working with the [!DNL Mailchimp Tags] destination.
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Mailchimp Tags] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+### Prerequisites in [!DNL Mailchimp Tags] {#prerequisites-destination}
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: I think there is a terminology confusion here. I don't think `Mailchimp Tags` is an actual name of a product, but please double check this with the team. Instead, my understanding is that tags are a feature of the Mailchimp platform.

So this heading would be ### Prerequisites in [!DNL Mailchimp], since customers need to log in to their Mailchimp account, where they can configure their tags.

Again, please double check if this is correct.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+| Credential | Description |
+| --- | --- |
+| `Username` | Your [!DNL Mailchimp] account username. |
+| `Password` | Your [!DNL Mailchimp] account password. |
+
+{style="table-layout:auto"}
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next you need to identify your Mailchimp domain, to do this log in to your [!DNL Mailchimp] account and check the URL in your browser.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Let's break this phrase up. 

Next you need to identify your Mailchimp domain. To do this, log in to your [!DNL Mailchimp] account and check the URL in your browser.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+You’ll notice a value similar to *`us19`.admin.mailchimp.com*, the `us19` prefix is the value you will require.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: You’ll notice a value similar to *`us19`.admin.mailchimp.com*. Write down the prefix (`us19` in this example). You will need this prefix when connecting to the destination.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+If you require guidance, refer to the [Mailchimp documentation](https://mailchimp.com/developer/marketing/guides/quick-start/#make-your-first-api-call).
+
+### Guardrails {#guardrails}
+
+Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) which details the limits imposed by the [!DNL Mailchimp] API.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) for detailed information about the limits imposed by the [!DNL Mailchimp] API.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+## Supported identities {#supported-identities}
+
+[!DNL Mailchimp] supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact Email Address | Mandatory |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Contact email address

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+>
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]**, search for [!DNL Mailchimp Tags]. Alternatively you can locate it under the **[!UICONTROL Email marketing]** category.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**. You will be navigated to the [!DNL Mailchimp Log in and authorize] page. Enter your [!DNL Mailchimp] account credentials and select [!DNL Log In].
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: You will be redirected to the [!DNL Mailchimp] authorization page.

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: "Enter your [!DNL Mailchimp] account credentials and select [!DNL Log In]."

I believe there is a screenshot missing here. The image below shows the Connect to destination part, not the Mailchimp log in part.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+![Platform UI screenshot showing how to authenticate to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-tags/authenticate-destination.png)
+
+Next, Select [!UICONTROL Allow] in the subsequent window to give permissions to the **Adobe Experience Platform** app to access your [!DNL Mailchimp] account. *You will need to do this only once*.
+
+![Mailchimp App screenshot confirmation popup to give permissions to the Experience Platform app access to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-tags/allow-app.png)
+
+If the details provided are valid, the UI displays a message: *You successfully connected to Mailchimp Tags account* message and a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: If the details provided are valid, the Platform UI displays the following message along with a **[!UICONTROL Connected]** status with a green check mark: *[!DNL You successfully connected to Mailchimp Tags account]*. You can then proceed to the next step.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/mailchimp-tags/destination-details.png)
+
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL Name]** | A name by which you will recognize this destination in the future. |
+| **[!UICONTROL Description]** | A description that will help you identify this destination in the future. |
+| **[!UICONTROL Data Center]** | Your [!DNL Mailchimp] account `Data Center`. Refer to the [Identify [!DNL Mailchimp] data center](#identify-data-center) section for any guidance. |
+| **[!UICONTROL Audience List ID]** | After you provide the value of your `Data Center` entries in this select dropdown are automatically populated with the audience names from your account. You need to select the audience of interest to which you will add or update the contact details from selected Platform segments. |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: After you enter your [!UICONTROL Data Center] prefix, this dropdown is automatically populated with the audience names from your account. Select the audience that you want to update with data from Platform.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Mailchimp Tags] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. 
+
+To correctly map your XDM fields to the [!DNL Mailchimp Tags] destination fields, follow the steps below:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: In the **[!UICONTROL Select source field]** screen, choose  **[!UICONTROL Select identity namespace]** and select the `Email` identity namespace.

Also, let's add a screenshot for this step.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 13]
```diff
@@ -0,0 +1,173 @@
...
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: In the **[!UICONTROL Select target field]** screen, choose **[!UICONTROL Select identity namespace]** and select the `Email` identity namespace.

Also, let's add a screenshot for this step.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 14]
```diff
@@ -0,0 +1,173 @@
...
+    An example with the above mappings is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/email-marketing/mailchimp-tags/mappings.png)
+
+When you have finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Log in to the [[!DNL Mailchimp]](https://login.mailchimp.com/) website. Then navigate to the **[!DNL Audience]** > **[!DNL All Contacts]** page and check if the contacts from the segment have been added and contacts within the segment have been updated with the segment name.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Log in to your [[!DNL Mailchimp]](https://login.mailchimp.com/) account.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+![Mailchimp UI screenshot showing the Audience page.](../../assets/catalog/email-marketing/mailchimp-tags/contacts.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+Refer to the [[!DNL Mailchimp] Errors](https://mailchimp.com/developer/marketing/docs/errors/) page for a comprehensive list of status and error codes with explanations.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Refer to the [!DNL Mailchimp] [errors page](https://mailchimp.com/developer/marketing/docs/errors/)

**Review by vfalon — CHANGES_REQUESTED** [review]
Hey @mohan-h ! Thanks for submitting the Mailchimp documentation. It looks great, just a few comments to address before we get it ready.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md (hunk 4 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/mailchimp-tags.md]: Correct `Mailchimp Tags` is not a product however here we are referring to prerequisites for the destination. Have added the word destination to make it clearer and read as `### Prerequisites for the [!DNL Mailchimp Tags] destination {#prerequisites-destination}`. Note the subsequent sub-heading `#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}` refers to `Mailchimp`.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md (hunk 1 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/mailchimp-tags.md:2]: It is `T`ags.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+
+To help you better understand how and when you should use the [!DNL Mailchimp Tags] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of an organization wants to broadcast an email based marketing campaign to a curated list of contacts. The lists of contact are received in batches from different offline sources and therefore need to be tracked. The team identifies an existing [!DNL Mailchimp] audience and starts building the Experience Platform segment's into which the contacts from each list are added. After sending these segments to [!DNL Mailchimp Tags] if any contacts do not exist in the selected [!DNL Mailchimp] audience they get added with an associated tag having the segment name the contact belongs to. If any contacts already exist in the [!DNL Mailchimp] audience a new tag with the name of the segment is added. As the labels are visible in [!DNL Mailchimp] the offline sources are easily identifiable. After the data is sent over to [!DNL Mailchimp] they send the marketing campaign email to the audience.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 21: "starts building the Experience Platform segment's "  ---> "starts building the Experience Platform audiences"

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: "The lists of contact are received" ---> "The lists of contacts is received"

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: "After sending these segments to [!DNL Mailchimp Tags] if any contacts do not exist in the selected [!DNL Mailchimp] audience they get added with an associated tag having the segment name the contact belongs to."

Please rephrase to:

"After sending these audiences to [!DNL Mailchimp Tags], if any contacts do not exist in the selected [!DNL Mailchimp] audience, they get added with an associated tag which includes the audience name the contact belongs to."

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 17]
```diff
@@ -0,0 +1,199 @@
...
+
+#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}
+
+Before you can create a [!DNL Mailchimp Tags] destination, you must first ensure that you have a [!DNL Mailchimp] account. If you do not have one already visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.
+
+#### Gather [!DNL Mailchimp] API key {#gather-credentials}
+
+You need your [!DNL Mailchimp] **API key** to authenticate the [!DNL Mailchimp Interest Categories] destination against your [!DNL Mailchimp] account. The **API key** serves as the **Password** when you [authenticate the destination](#authenticate).
+
+If you do not have your **API key**, Sign in to your account and refer to the [[!DNL Mailchimp] Generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key) documentation to create one.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 43: lowercase s in "Sign in"

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Please reword this to:

If you do not have your **API key**, sign in to your [!DNL Mailchimp] account and refer to the [!DNL Mailchimp] documentation on [how to generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key).

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 18]
```diff
@@ -0,0 +1,199 @@
...
+
+If you do not have your **API key**, Sign in to your account and refer to the [[!DNL Mailchimp] Generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key) documentation to create one. 
+
+An example of an API key is `0123456789abcdef0123456789abcde-us14`. 
+
+>[!IMPORTANT]
+>
+>If you generate the **API key**, write it down as you will not be able to access it after generation.
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 51: Identify your [!DNL Mailchimp] data center {#identify-data-center}

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 19]
```diff
@@ -0,0 +1,199 @@
...
+
+Next, you must identify your [!DNL Mailchimp] data center. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 55: The data center ID is the first section of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center ID is `us14`.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 20]
```diff
@@ -0,0 +1,199 @@
...
+>
+>If you generate the **API key**, write it down as you will not be able to access it after generation.
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next, you must identify your [!DNL Mailchimp] data center. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`. 
+
+It's also appended to your API key in the form *key-dc*; if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 57: The data center ID is also appended to your API key in the form *key-dc*. For example, if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 21]
```diff
@@ -0,0 +1,199 @@
...
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next, you must identify your [!DNL Mailchimp] data center. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`. 
+
+It's also appended to your API key in the form *key-dc*; if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`. 
+
+Write down the data center value *(`us14` in this example)*, you need this value when you [fill in destination details](#destination-details).
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 59: Write down the data center value *(`us14` in this example)*. You will need this value when you [fill in the destination details](#destination-details).

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 22]
```diff
@@ -0,0 +1,199 @@
...
+
+Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) for detailed information about the limits imposed by the [!DNL Mailchimp] API.
+
+## Supported identities {#supported-identities}
+
+[!DNL Mailchimp] supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact email Address | Mandatory |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 73: Update the description to: The email address of the contact.

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 23]
```diff
@@ -0,0 +1,199 @@
...
+
+{style="table-layout:auto"}
+
+## Supported audiences {#supported-audiences}
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md:81]: Please update the supported audiences table to:


| Audience origin | Supported | Description | 
---------|----------|----------|
| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
| Custom uploads | X | Audiences [imported](../../../segmentation/ui/overview.md#import-audience) into Experience Platform from CSV files. |

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 24]
```diff
@@ -0,0 +1,199 @@
...
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> For each selected segment in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with its segment status from Platform.</li></ul>|
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Line 93: "You are exporting all members of a segment, " --> "You are exporting all members of an audience, "

Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: "For each selected segment in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with its segment status from Platform." --> "For each audience selected in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with the audience status from Platform."

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 25]
```diff
@@ -0,0 +1,199 @@
...
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: "As soon as a profile is updated in Experience Platform based on segment evaluation,"  --> "As soon as a profile is updated in Experience Platform based on audience evaluation,"

File: help/destinations/catalog/email-marketing/mailchimp-tags.md [hunk 26]
```diff
@@ -0,0 +1,199 @@
...
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]**, search for [!DNL Mailchimp Tags]. Alternatively you can locate it under the **[!UICONTROL Email marketing]** category.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields below and select **[!UICONTROL Connect to destination]**.
+
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Mailchimp Interest Categories] username. |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/mailchimp-tags.md]: Your [!DNL Mailchimp Interest Categories] username.

I believe this should say Mailchimp Tags.

**Review by vfalon — CHANGES_REQUESTED** [review]
@mohan-h I've reviewed the latest changes and left some more comments to address. Please let me know if you have any questions. Thank you!
